### PR TITLE
feat(readonlyvalue): Read only value updates

### PR DESCRIPTION
**Pull request by cgirani**
**Summary**

- Support to send a node in prop value of ReadOnlyValue  component

**Change List (commits, features, bugs, etc)**

- Prop support to render a node

**Acceptance Test (how to verify the PR)**

- Check that the value prop supports a node to be rendered and 

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Nothing changed in the already supported cases

**Things to look for during review**

- [ ] Make sure all references to `iot` or `bx` class prefix is using the prefix variable
- [ ] (React) All major areas have a `data-testid` attribute. New test ids should have test written to ensure they are not changed or removed.
- [ ] UI should be checked in RTL mode to ensure the proper handling of layout and text.
- [ ] All strings should be translatable.
- [ ] The code should pass a11y scans (The storybook a11y knob should show no violations). New components should have a11y test written.
- [ ] Unit test should be written and should have a coverage of 90% or higher in all areas.
- [ ] All components should be passing visual regression test. For new styles or components either a visual regression test should be written for all permutations or the base image updated.
- [ ] Changes or new components should either write new or update existing documentation.
- [ ] PR should link and close out an existing issue

### CODE REVIEW

File: packages/react/src/components/ReadOnlyValue/_read-only-value.scss [hunk 1]
```diff
@@ -2,7 +2,8 @@
   display: block;
   margin-bottom: $spacing-06;
   label,
-  textarea {
+  textarea,
+  div {
```
Comment (davidicus) [inline packages/react/src/components/ReadOnlyValue/_read-only-value.scss:6]: this only works if they pass in a div. Should we wrap the node content in a div to ensure this style is applied?

**Review by davidicus — COMMENTED** [review]
Looks good. Just the one comment on the styling of the div.

File: packages/react/src/components/ReadOnlyValue/_read-only-value.scss (hunk 1 above)
Comment (cgirani) [reply packages/react/src/components/ReadOnlyValue/_read-only-value.scss:6]: Actually, i'm wrapping the value prop into the div, at the code... check line 44 in `ReadOnlyValue.jsx`

Comment (davidicus) [reply packages/react/src/components/ReadOnlyValue/_read-only-value.scss:6]: You are right. My mistake!

**Review by davidicus — APPROVED** [review]

**Pull request merged**